go/vt/vtgate/evalengine/compiler_test.go

```diff
@@ -478,6 +480,7 @@ func TestCompilerSingle(t *testing.T) {
 			}
 
 			env := evalengine.EmptyExpressionEnv()
+			env.SetTime(time.Date(2023, 10, 24, 12, 0, 0, 0, tz))
```

Comment:
@dbussink: the tests we wrote yesterday break if they're not run yesterday :sweat_smile: